internal/git/git_test.go

```
@@ -0,0 +1,209 @@
+
+	// Save current directory to restore later
+	currentDir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get current directory: %v", err)
+	}
+
+	// Change to repo directory
+	if err := os.Chdir(repoDir); err != nil {
+		t.Fatalf("Failed to change directory to %s: %v", repoDir, err)
+	}
+	defer os.Chdir(currentDir)
```

golangci-lint (1.64.8)

Error return value of `os.Chdir` is not checked

(errcheck)